src/app/space-wizard/services/fabric8-app-generator.service.ts

```diff
@@ -336,7 +336,8 @@ export class Fabric8AppGeneratorService extends AppGeneratorService {
         return FieldWidgetClassificationOptions.SingleInput;
       }
       case 'uiselectone': {
-        return FieldWidgetClassificationOptions.SingleSelectionDropdown;
+        return FieldWidgetClassificationOptions.FilteredDropdown;
+        //return FieldWidgetClassificationOptions.SingleSelectionDropdown;
```

Comment:
@mitchharpur not sure how to hook to be ale to display the 2 different Dropdowns. We can't just be based on forge reponse. or do we want all dropdown to be filterable?